### PR TITLE
Use image size after processing as the resulting file size for image

### PR DESCRIPTION
**Pull request by rikukissa**
…which can be used for total file size limit calculation

### CODE REVIEW

File: packages/client/src/components/form/DocumentUploadField/DocumentUploaderWithOption.tsx [hunk 1]
```diff
@@ -152,26 +152,26 @@ export const DocumentUploaderWithOption = (props: IFullProps) => {
...
-    // disable compression with a falsy value
-    const resized =
-      Boolean(options.maxSizeMB) &&
-      bytesToMB(uploadedImage.size) > options.maxSizeMB &&
-      (await imageCompression(uploadedImage, options))
+    if (
+      !Boolean(options.maxSizeMB) ||
+      bytesToMB(uploadedImage.size) <= options.maxSizeMB
+    ) {
+      return uploadedImage
```
Comment (tahmidrahman-dsi) [inline packages/client/src/components/form/DocumentUploadField/DocumentUploaderWithOption.tsx:159]: much nicer than previously assigned processed image to the output variable

File: packages/client/package.json [hunk 2]
```diff
@@ -111,6 +111,7 @@
     "@graphql-codegen/introspection": "^3.0.0",
     "@graphql-codegen/typescript": "^3.0.0",
     "@graphql-codegen/typescript-operations": "^3.0.0",
+    "@types/browser-image-compression": "^1.0.13",
```
Comment (tahmidrahman-dsi) [inline packages/client/package.json:114]: Thank you! 🙏🏼

**Review by tahmidrahman-dsi — APPROVED** [review]
LGTM

**Review by tahmidrahman-dsi — APPROVED** [review]

**Comment by rikukissa** [comment]
E2Es passed here: https://github.com/opencrvs/e2e/actions/runs/11796515581/job/32858703302

**Pull request merged**